slides.md

```diff
@@ -401,7 +402,7 @@ export async function comparePassword(password, hash) {
 <div class="dense">
 
 - Run the server for step 3 (`cd src/a03-injection`, `npm start`)
-- In Postman, run the query for `A03: Get customer by name`. Observe the data for `name: "alice"` being returned
+- From the browser at `http://localhost:3006`, run the query for `A03: Get customer by name`. Observe the data for `name: "alice"` being returned
```

Comment:
this exercise returns a weird error on the browser:
<img width="1920" alt="Screenshot 2023-10-16 at 12 23 15" src="https://github.com/nearform/owasp-top-ten-workshop/assets/36735501/7d02e0aa-49e5-49cf-8039-35dc4cdce9cd">

Reply:
Actually DB error:
<img width="1920" alt="Screenshot 2023-10-16 at 12 25 25" src="https://github.com/nearform/owasp-top-ten-workshop/assets/36735501/ef975aea-3f82-4a5c-befb-4bbe0f088d8c">